scripts/generator/graphql_generator/csharp/Cart.cs.erb

```diff
@@ -191,218 +152,60 @@ namespace <%= namespace %> {
-                        if (CurrentCheckout.ready()) {
-                            callback(null);
-                        } else {
-                            PollCheckoutAndUpdate(PollCheckoutAvailableShippingRatesReady, callback);
-                        }
-                    } else {
-                        HandleUserError(callback);
-                    }
-                }
-            });
-        }
-
-        public void SetShippingAddress(MailingAddressInput mailingAddressInput, CompletionCallback callback) {
-            MutationQuery query = new MutationQuery();
-
-            DefaultQueries.checkout.ShippingAddressUpdate(query, CurrentCheckout.id(), mailingAddressInput);
-
-            Client.Mutation(query, (Mutation response, ShopifyError error) => {
-                if (error != null) {
-                    callback(error);
-                } else {
-                    if (UpdateState(response.checkoutShippingAddressUpdate().checkout(), response.checkoutShippingAddressUpdate().userErrors())) {
-                        PollCheckoutAndUpdate(PollCheckoutAvailableShippingRatesReady, callback);
-                    } else {
-                        HandleUserError(callback);
-                    }
-                }
-            });
-        }
-
-        public void SetFinalCheckoutFields(string email, ShippingFields? shippingFields, CompletionCallback callback) {
-            MutationQuery query = new MutationQuery();
-
-            DefaultQueries.checkout.EmailUpdate(query, CurrentCheckout.id(), email);
-
-            if (shippingFields.HasValue) {
-                DefaultQueries.checkout.ShippingAddressUpdate(query, CurrentCheckout.id(), shippingFields.Value.ShippingAddress);
-                DefaultQueries.checkout.ShippingLineUpdate(query, CurrentCheckout.id(), shippingFields.Value.ShippingIdentifier);
-            }
-
-            Client.Mutation(query, (Mutation response, ShopifyError error) => {
-                if (error != null) {
-                    callback(error);
-                } else {
-                    var userErrors = response.checkoutShippingAddressUpdate().userErrors();
-                    if (shippingFields.HasValue) {
-                        userErrors.AddRange(response.checkoutShippingLineUpdate().userErrors());
-                        userErrors.AddRange(response.checkoutEmailUpdate().userErrors());
-                    }
-
-                    if (UpdateState(response.checkoutEmailUpdate().checkout(), userErrors)) {
-                        PollCheckoutAndUpdate(PollCheckoutReady, callback);
-                    } else {
-                        HandleUserError(callback);
-                    }
-                }
-            });
-        }
-
-        private void CheckoutSave(CompletionCallback callback) {
+        public void CheckoutSave(CompletionCallback callback) {
```

Comment:
It might make more sense - I was just wary about changing the public facing API name especially if people are using it.